### ISSUE TITLE
[bug]: The docs expandable in the arguments table are not focusable

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5259]
### 🐞 Describe the bug

The expandable elements in the documentation's arguments table are not focusable, therefore we can't see its content using keyboard navigation.
The issue can be seen in the [Components / Accordion docs](https://next.design-system.post.ch/?path=/docs/4d1b4185-e04d-494a-ab38-2b56c1778b0b--docs).

### 📷 Screenshots

![Image](https://github.com/user-attachments/assets/d868d83e-6b0e-4f8f-804d-84dc06592da7)

**Comment by leagrdv** [comment]
The issue is within storybook and is not fixable with just config. I've opened a ticket for it on their repo: [Link to the ticket](https://github.com/storybookjs/storybook/issues/31436)
